README.md

```diff
@@ -0,0 +1,93 @@
+# Spotify Web SDK
+
+A JavaScript SDK for the [Spotify Web API](https://developer.spotify.com/documentation/web-api/).
+
+## Features
+
+### "Why should I use this?"
+
+We want to handle the hard work for you.
+Even to make simple requests, such as to get information on a track, you'd need to make a bunch of setups.
+Here is how you could do the same thing with our aid:
+
+```javascript
+import { init, getTrack } from 'spotify-web-sdk';
+init('YOUR SPOTIFY TOKEN HERE!'); // You should only need to run this once.
+getTrack('3LOpHuEpjkL4T1Zcjhko8w'); // Or any other track id.
+```
+
+Yes, it's as simple as that!
+
+### "I still think it's no hard work at all. Therefore, I don't see the point in using this."
+
+How rude!
+As you might have noticed, this is an open source project, which means that you can add or suggest your own features.
+Of course, don't expect us to ever have a method such as `getTaylorSwiftBestTracks` pushed into our master branch.
+(But if you ever thought on something like that, hey, let's be friends!)
+
+What we really mean is: there's a lot of information that can be extracted from the data retrievable via Spotify's Web API.
+If you think of something that could come in handy for you and also other people, that's it! Don't be shy, make it happen.
+We'll gladly review your suggestion as soon as possible!
+
+For instance, you can retrieve the duration of a track in milliseconds straight from the API, but know what'd more usual?
+If you thought "displaying it in minutes", we're on the same page.
+To do that, adding the following code to our Track model would do the trick:
+
+```javascript
+get formattedDuration() {
+    const minutes = Math.floor(this.durationMs / 60000);
+    const seconds = Math.floor((this.durationMs % 60000) / 1000);
+    return `${minutes}:${seconds < 10 ? '0' : ''}${seconds}`;
+}
+```
+
+This is already implemented, but do you get our point?
+It might be something really simple, but we believe that such little things can save a lot of time in the future.
+
+### "You convinced me. Let's go!"
+
+Even though I'm not certain you thought that right now, here's how to get started:
+
+## Getting started
+
+### Installation
```

Comment:
Don't you think better to put the `installation` and `usage` sections before the `features` section?